### ISSUE TITLE
[Suggestion] Transmutation Harvester

### ISSUE THREAD

**Issue by Vercalos** [latvian-dev/ProjectEX#35]
Just had an idea; what if you could make a tool that acted in a similar fashion to an EMC link.  It would harvest items normally, but if it had an EMC value, it would convert the item into EMC instead of adding it to your inventory.

**Comment by GigaOtomia** [comment]
I imagine this could be a combination of a Gem of Eternal Density and some tier of EMC Link. Reuse some code, tie it to your personal EMC count, draw a small X on the gem and call it a day.

**Comment by Vercalos** [comment]
Nah.  I'm thinking of something more along the lines of a pickaxe or something that dumps EMC stuff right into the network.  The Gem of Eternal Density is a bit of an annoyance to set up the blacklists so all my food and torches aren't eaten by it.
